examples/chat/components/MessageSection.vue

```diff
@@ -1,54 +1,54 @@
 <template>
   <div class="message-section">
     <h3 class="message-thread-heading">{{ thread.name }}</h3>
-    <ul class="message-list" ref="list">
-      <message
-        v-for="message in sortedMessages"
+    <ul class="message-list"
+      ref="list">
+      <message v-for="message in messages"
```

Comment:
Can we revert the format change? This looks unnecessary and the previous format would readable and generic, IMO.